### PR TITLE
fix bugs to make fluent bit work on ubuntu

### PR DESCRIPTION
**Pull request by anishk25**
Had to make some changes to the fluent bit formula to make it properly work

### CODE REVIEW

File: fluent-bit/defaults.yaml [hunk 1]
```diff
@@ -10,7 +10,6 @@ fluent_bit:
     flush: 5
     daemon: 'Off'
     log_level: info
-    parsers_file: parsers.conf
```
Comment (vnllgrlla) [inline fluent-bit/defaults.yaml:13]: Why remove?

Comment (anishk25) [reply fluent-bit/defaults.yaml:13]: This was causing the parsers to be written twice to the config and causing errors when fluent bit started up. I specify the parsers in the pillar file so this is not needed here.

**Review by snoshy — APPROVED** [review]

File: fluent-bit/defaults.yaml (hunk 1 above)
Comment (vnllgrlla) [inline fluent-bit/defaults.yaml:13]: why remove this? do we manually configure parsers somewhere or is fluent bit just a forwarder?

Comment (anishk25) [reply fluent-bit/defaults.yaml:13]: This was causing the parsers to be written twice to the config and causing errors when fluent bit started up. I specify the parsers in the pillar file so this is not needed here.

**Pull request merged**